### PR TITLE
Rules fp fixes 2021 02

### PR DESCRIPTION
**Pull request by mstemm**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

/kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area engine

/area rules

> /area tests

> /area proposals

**What this PR does / why we need it**:

Address various rules FPs seen in real-world environments.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Address various rules FPs seen in real-world environments.
```

### CODE REVIEW

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -2417,6 +2421,8 @@
     - name: proc_name_image_suffix
       fields: [proc.name, container.image.repository]
       comps: [in, endswith]
+      values:
+        - [[rpm], quay.io/sysdig/node-image-analyzer]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:2425]: @mstemm , I think `node-image-analyzer` will go to the customization on our side, not in OSS

File: rules/falco_rules.yaml [hunk 2]
```diff
@@ -1537,6 +1539,8 @@
         - [[rancher-bridge], "rancher/network-manager"]
         - [[calico-node], "calico/node"]
         - [[scope], "weaveworks/scope"]
+        - [[nsenter], "quay.io/cilium/startup-script"]
+        - [[java], "docker.io/sysdig/agent"]
```
Comment (Kaizhe) [inline rules/falco_rules.yaml:1543]: agent will go to the customization.

**Review by Kaizhe — CHANGES_REQUESTED** [review]

File: rules/falco_rules.yaml (hunk 1 above)
Comment (leodido) [reply rules/falco_rules.yaml:2425]: Agree

File: rules/falco_rules.yaml (hunk 2 above)
Comment (leodido) [reply rules/falco_rules.yaml:1543]: Agree with Kaizhe

**Pull request closed without merging**